test/file.ts

```diff
@@ -2617,6 +2617,21 @@ describe('File', () => {
         );
       });
 
+      it('should set correct settings if resumable', done => {
```

Comment:
Please use `async` for new tests :)

Reply:
the unit tests mock out `promisify`.... I'll try!

Reply:
ah, no big deal if it's a pain, just a preference :)